GOVERNANCE.md

```diff
@@ -1,88 +1,234 @@
+
+- Provide support for Backstage by removing blockers, securing funding, providing advocacy, feedback, and ensuring project continuity and long term success
+- Assist Backstage maintainers in prioritizing upcoming roadmap items and planned work
+- Provide neutral mediation for any disputes that arise as part of the project
+
+### Backstage End User Sponsor Membership
 
 The End User Sponsors group comprises at most 5 people. To be eligible for membership in the group, you or the company where you work you must:
 
 - Be responsible for and end user of a production Backstage deployment of non-trivial size
 - Be active contributors to the open source project
 - Be willing and able to attend regularly-scheduled End User Sponsor meetings
-- Abide by [Backstage’s Code of Conduct](./CODE_OF_CONDUCT.md).
+- Abide by [CNCF CoC](https://github.com/cncf/foundation/blob/main/code-of-conduct.md)
 
 Candidates for membership will be nominated by current Sponsor members or by Backstage maintainers. If there are more nominations than Sponsor seats remaining, existing sponsors shall vote on the candidates, and the candidates with the most votes will become Sponsors. Any ties will be broken by current Backstage sponsors.
 
-# Reviewers
+# Conflict resolution and voting
 
-The project also contains a team called [@backstage/reviewers](https://github.com/orgs/backstage/teams/reviewers). This is the team of people who are the fallback in [`CODEOWNERS`](./.github/CODEOWNERS). This team will typically contain the maintainers, and a small number of additional people who are permitted to approve and merge pull requests. The purpose of this group is to offload some of the review work from the maintainers, simplifying and speeding up the review process for contributors.
+In general, we prefer that technical issues and membership are amicably worked out between the persons involved. If a dispute cannot be decided independently, the sponsors and core maintainers can be called in to decide an issue. If the sponsors and maintainers themselves cannot decide an issue, the issue will be resolved by voting.
 
-This responsibility is distinct from the maintainer role. A reviewer must not approve and merge changes that have a level of impact that a maintainer should oversee; see below for clarification. For that class of changes, a reviewer can still review the pull request thoroughly without approving it (e.g. with a comment on the pull request), and is expected to notify `@backstage/maintainers` for final approval. Note that it is best to not use the GitHub review approve functionality for this, since that would let Hall of Fame members self-merge the pull request before maintainers get the chance to look at it.
+In all cases in this document where voting is mentioned, the voting process is a simple majority in which each sponsor receives two votes and each maintainer receives one vote. If such a majority is reached, the vote is said to have _passed_.
```

Comment:
1. Presumably this means Core Maintainers and End-User Sponsors
2. Majority of what?  Majority of all CMs and Sponsors in existence?  or majority of those voting?  (I recommend the former).
3. This gives the End-User Sponsors as a group much more authority than the CMs, despite the CMs having much more stringent requirements for their status.  Was that intentional?

Reply:
Also:
4. Where do votes take place?